examples/pytorch/text-generation/run_generation.py

```diff
@@ -355,7 +354,9 @@ def main():
     if tokenizer.pad_token is None:
         tokenizer.pad_token = tokenizer.eos_token
     model = model_class.from_pretrained(args.model_name_or_path)
-    model.to(args.device)
+
+    # Prepare model with `accelerator`.
+    accelerator.prepare_model(model)
 
     if args.fp16:
         model.half()
```

Comment:
You can probably use Accelerate's mixed precision here, pass it into `Accelerator(mixed_precision={})` and it will convert it ootb

Reply:
This is eval only in this script. There is no mixed precision.

Reply:
Oops, good point. Disregard!